src/test/java/api/SiteApi.java

```diff
@@ -0,0 +1,28 @@
+package api;
+
+
+import com.jayway.restassured.RestAssured;
+
+
+public class SiteApi extends RestAssured {
+    public SiteApi() {
+        this("https://www.treatwell.co.uk", "api");
+    }
+
+    public SiteApi(String baseURI, String basePath) {
+        RestAssured.baseURI = baseURI;
+        RestAssured.basePath = basePath;
+    }
+
+
+    public String post(String body, String specificPath, int statusCode) {
+         return given().contentType("application/json").body(body).
+                 when().post(specificPath).
+                 then().statusCode(statusCode).
+                 extract().response().asString();
```

Comment:
This code is indented differently to how I'd expect. When chaining functions over multiple lines, acceptable indentation is either all the lines with the same starting column, e.g.

```java
return line1.
       line2.
       line3;
```

...or all the lines indented by 4, e.g.

```java
return 
    line1.
    line2.
    line3;
```